### ISSUE TITLE
[Feature Request] UI Artifact support for .log files as text

### ISSUE THREAD

**Issue by vfdev-5** [mlflow/mlflow#375]
It would be great to be able to read in UI the logged artifact files with `log` extension as text files.

More details, during the training a log file `train.log` is written at a temp folder and finally logged as artifact to mlflow. However, in the UI frontend it is not possible to read its content. Message  "Supported formats: image and text files" is shown. 

Let me know what you think about this feature ?
Thanks

**Comment by mparkhe** [comment]
That's a great idea! Would you like to submit a Pull Request for this? I would make sure to handle a case of a file with `.txt` extension and directory named `log`.

cc: @mateiz  and @andrewmchen for thoughts on this for support in js.

**Comment by mateiz** [comment]
Yup, it should be easy to add log into the TEXT_EXTENSIONS list in the server and frontend. Search for "md", "rst" etc in there to find it.
